src/Select.soy

```diff
@@ -39,7 +39,7 @@
 			{param expanded: $disabled ? false : $expanded_ /}
 			{param header kind="html"}
 				<button
-					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					class="{$buttonClass ?: 'btn btn-default'} dropdown-select"
```

Comment:
We could remove it, and it would render correctly. However it might be good to leave it since it makes the component's default behavior more obvious when looking at the JS implementation.